src/components/BackOffice/CloseWorkOrders/index.js

```diff
@@ -88,7 +88,39 @@ const CloseWorkOrders = () => {
     return newErrors
   }
 
-  const handleSubmit = (event) => {
+  const renderConfirmationModal = () => {
```

Comment:
Remember there are instances where 300+ work orders are being closed at the same time

Reply:
Thanks, might be worth removing the list of work orders!